.puppet/profiles/icingaweb2_dev/manifests/init.pp

```diff
@@ -21,7 +21,16 @@
   }
 
   # TODO(el): icinga-gui is not a icingaweb2_dev package
-  package { [ 'php-gd', 'php-intl', 'php-pdo', 'php-ldap', 'php-phpunit-PHPUnit', 'icinga-gui' ]:
+  package { [
+    'rh-php71-php-gd',
+    'rh-php71-php-intl',
+    'rh-php71-php-pdo',
+    'rh-php71-php-mysqlnd',
+    'rh-php71-php-pgsql',
+    'rh-php71-php-ldap',
+    'php-phpunit-PHPUnit',
+    'icinga-gui'
```

Comment:
You can drop this package, that's the Classic UI which isn't available as meta package for Icinga 2 anymore.

Reply:
#3254